src/driver.c

```diff
@@ -996,15 +1002,15 @@ main (int ac, char **av)
 	d_path = NULL;
 	
 	process_options (ac, av);
-#if ( defined(WIN32) && !defined(_POSIX_C_SOURCE) )
+/*#if ( defined(WIN32) && !defined(_POSIX_C_SOURCE) )
```

Comment:
If you're dropping a piece of code, just drop it. Exceptions to this rule require a good reason. Again, remember we have everything saved up in previous commits, the text is not going away in case we need it back sometime.

Reply:
It's a very strange piece of code. As far as I understand, it's necessary for parallel data generation on Windows, but it's not implemented and currently, this code has no purpose. Should I delete it in such situation?

Reply:
You mean, the code after the `#if` has no purpose?

Anyway, if it's not causing any problems, I'd leave it as-is.